.github/workflows/bench.yaml

```diff
@@ -0,0 +1,55 @@
+name: Benchmarks
+
+permissions:
+  contents: read
+
+on:
+  pull_request:
+
+jobs:
+  bench:
+    runs-on: ubuntu-22.04
+    steps:
+      - uses: actions/checkout@v3
+      - uses: actions/checkout@v3
+        with:
+          ref: main
+          path: baseline-branch
+
+      - name: Prepare Environment
+        run: |
+          rustup update stable
+          rustup default stable
+          cargo install cargo-export --version 0.2.0
+
+      - uses: actions/cache@v3
+        with:
+          path: |
+            ~/.cargo/bin/
+            ~/.cargo/registry/index/
+            ~/.cargo/registry/cache/
+            ~/.cargo/git/db/
+            ./target/
+            ./baseline-branch/target/
+          key: Bench/${{ runner.os }}
+
+      - name: Building Benchmarks
+        run: |
+          cargo export target/benchmarks -- bench --bench=search_ord
+          # !!!SHOULD BE FIXED AFTER MERGE TO MAIN!!!
+          # At the moment we can only test branch agains itself, because there is no
+          # tango benchmarks on the main.
+          # cd baseline-branch
+          # cargo export target/benchmarks -- bench --bench=search_ord
+
+      - name: Run Benchmarks
+        run: |
+          set -eo pipefail
+
+          target/benchmarks/search_ord --color=never compare target/benchmarks/search_ord \
+            -ot 1 --fail-threshold 10 | tee target/benchmark.txt
```

Comment:
What is `-ot` here?
Also, what is the unit for `--fail-threshold`?

Reply:
- `-t` – duration of each test (in seconds)
- `-o` – filter outliers. Because tango is measuring difference directly it can detect and remove severe observations that are manifest itself symmetrically (in both algorithms), therefore they are considered independent of the algorithms performance.
- `--fail-threshold` – exit with non zero exit code if candidate is slower than baseline on given amount of percents. Tango also do a z-test and fail only if difference is statistically significant (planning to move to bootstrap later on)

More on [cli-arguments](https://github.com/bazhenov/tango?tab=readme-ov-file#runner-arguments)